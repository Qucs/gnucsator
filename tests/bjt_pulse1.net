
BJT:Q2N3904_1 in c out c Type="npn" Is="1.4e-14" Nf="1" Nr="1" Ikf="0.025" Ikr="0" Vaf="100" Var="0" Ise="3e-13" Ne="1.5" Isc="0" Nc="2" Bf="300" Br="7.5" Rbm="0" Irb="0" Rc="2.4" Re="0" Rb="0" Cje="4.5e-12" Vje="0.75" Mje="0.33" Cjc="3.5e-12" Vjc="0.75" Mjc="0.33" Xcjc="1" Cjs="0" Vjs="0.75" Mjs="0" Fc="0.5" Tf="4e-10" Xtf="0" Vtf="0" Itf="0" Tr="2.1e-08" Temp="26.85" Kf="9e-16" Af="1" Ffe="1" Kb="0" Ab="1" Fb="1" Ptf="0" Xtb="1.5" Xti="3" Eg="1.11" Tnom="26.85" Area="1"
Vdc:V1 c gnd U="5 V"
R:R3 gnd out R="100 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
Relais:XX gnd out gnd gnd Vt="1.2" Vh="1"
Vpulse:V2 in gnd U1="0 V" U2="3 V" T1="0" T2="1 ms" Tr=".1 ms" Tf=".1 ms"
.TR:TR1 Type="lin" Start="0" Stop="1.5 ms" Points="1000" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1e-16" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
