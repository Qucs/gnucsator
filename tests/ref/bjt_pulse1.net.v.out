
BJT #(.Ab(1),.Af(1),.Area(1),.Bf(300),.Br(7.5),.Cjc(3.5e-12),.Cje(4.5e-12),.Cjs(0),.Eg(1.11),.Fb(1),.Fc(0.5),.Ffe(1),.Ikf(0.025),.Ikr(0),.Irb(0),.Is(1.4e-14),.Isc(0),.Ise(3e-13),.Itf(0),.Kb(0),.Kf(9e-16),.Mjc(0.33),.Mje(0.33),.Mjs(0),.Nc(2),.Ne(1.5),.Nf(1),.Nr(1),.Ptf(0),.Rb(0),.Rbm(0),.Rc(2.4),.Re(0),.Temp(26.85),.Tf(4e-10),.Tnom(26.85),.Tr(2.1e-08),.Type(npn),.Vaf(100),.Var(0),.Vjc(0.75),.Vje(0.75),.Vjs(0.75),.Vtf(0),.Xcjc(1),.Xtb(1.5),.Xtf(0),.Xti(3)) Q2N3904_1 (.b(in),.c(c),.e(out),.s(c));
Vdc #(.U(5)) V1 (.p(c),.n(0));
R #(.R(100),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85)) R3 (.p(0),.n(out));
Relais #(.Vh(1),.Vt(1.1)) XX (.1(0),.2(out),.3(0),.4(0));
Vpulse #(.U1(0),.U2(3),.T2(1m),.TL(NA( NA)),.Tr(.1m),.Tf(.1m),.T1(0)) V2 (.p(in),.n(0));
